recipe_scrapers/_utils.py

```diff
@@ -72,6 +72,10 @@
 )
 
 
+def format_diet_name(diet_name):
+    return " ".join(word for word in re.findall("[A-Z][^A-Z]*", diet_name))
```

Comment:
Could we remove this reformatting/filtering?  It seems like it would prevent data from non-Latin script languages from being returned.

Reply:
Made some adjustments that should remove the possibility of unintended cleaning. This is to handle instances where the input is formatted as 

```
"suitableForDiet":["https://schema.org/LowFatDiet"]
```
returning
```
Low Fat Diet
```

<details>
  <summary>Covers all RestrictedDiet outputs</summary>
```
[DiabeticDiet]
[GlutenFreeDiet]
[HalalDiet]
[HinduDiet]
[KosherDiet]
[LowCalorieDiet]
[LowFatDiet]
[LowLactoseDiet]
[LowSaltDiet]
[VeganDiet]
[VegetarianDiet]
```
</details>